Gruntfile.js

```diff
@@ -8,6 +8,14 @@ module.exports = function(grunt) {
 	var pullRequest = process.env.TRAVIS_PULL_REQUEST;
 
 	grunt.initConfig({
+	    eslint: {
+			target: ['Source/**/*.js'],
+			options: {
+				configFile: 'Tests/eslint-configure/eslint.json',
```

Comment:
If you use .eslintrc, the file is automatically picked up by the command line tool, which is useful for editor plugins for example.